R/relational.R

```diff
@@ -133,6 +133,19 @@ rel_project <- function(rel, exprs) {
   rethrow_rapi_rel_project(rel, exprs)
 }
 
+#' Lazily project a DuckDB relation object
+#' @param rel the DuckDB relation object
+#' @param exprs a list of DuckDB expressions to project
+#' @return the now projected `duckdb_relation` object
+#' @noRd
+#' @examples
+#' con <- DBI::dbConnect(duckdb())
+#' rel <- rel_from_df(con, mtcars)
+#' rel2 <- rel_project(rel, list(expr_reference("cyl"), expr_reference("disp")))
+rel_project2 <- function(df, con, exprs) {
+  rethrow_rapi_rel_project2(as.data.frame(df), con@conn_ref, exprs)
```

Comment:
This should be sufficient:

```suggestion
  rethrow_rapi_rel_project2(df, con@conn_ref, exprs)
```

Reply:
Why does this still show up here?